### ISSUE TITLE
Make the buttons make more sense

### ISSUE THREAD

**Issue by vinport16** [vinport16/KHE18#14]
right now, when you hit resume, it grays out all the buttons, but when you select a building while the clock is running, you can still upgrade them, which makes sense. Make it so the selected tower specific buttons aren't disabled upon resuming the game.

(and maybe let people place buildings while the clock is running also? probably would be fine tbh)

**Comment by mes258** [comment]
Agreed - players should be able to place buildings, upgrade, do whatever while the game is playing.

**Comment by mes258** [comment]
Players can now add buildings while the game is not paused.
